[T4A3][W10-B3] Matthew Chan

MCHowl opened the pull request.

MCHowl committed:
[T3A2][W10-A1] Matthew Chan
Fix encapsualtion within CommandResult

MCHowl committed:
[T3A3][W10-A1] Matthew Chan
Split Address into 4 subclasses
Add Block
Add Street
Add Unit
Add PostalCode

MCHowl committed:
[T3A4][W10-A1] Matthew Chan
Add catch for non-existent storage file upon save
Note: Unable to replicate bug where program would crash if the storage file is deleted while program is running.

MCHowl commented:
ready for review

mauris commented:
@MCHowl please follow the title format strictly, and submit only one activity per pull request.

MCHowl committed:
[T4A3][W10-B3] Matthew Chan
Add Contact as parent class
Remove repeated functions from Address
Remove repeated functions from Email
Remove repeated functions from Phone

damithc commented:
PR not clean. Closing without review.
